src/stm32/spi.c

```diff
@@ -21,14 +21,17 @@ DECL_ENUMERATION("spi_bus", "spi1", 1);
 DECL_CONSTANT_STR("BUS_PINS_spi1", "PA6,PA7,PA5");
 DECL_ENUMERATION("spi_bus", "spi1a", 2);
 DECL_CONSTANT_STR("BUS_PINS_spi1a", "PB4,PB5,PB3");
-#if !CONFIG_MACH_STM32F1
+#if CONFIG_MACH_STM32G4
+ DECL_ENUMERATION("spi_bus", "spi2a", 3);
+ DECL_CONSTANT_STR("BUS_PINS_spi2a", "PA10,PA11,PF1");
+#elif !CONFIG_MACH_STM32F1
```

Comment:
If a new definition is needed, please use the new format - `spi2_PA10_PA11_PF1`.